### ISSUE TITLE
Check dependency 

### ISSUE THREAD

**Issue by antuz123** [bithyve/bitcointribe#1493]
![92873A32-8013-4843-8D2B-5D862DBC9D59](https://user-images.githubusercontent.com/43998049/92449705-0f4a9f00-f1d8-11ea-80b0-0189d55beb75.jpeg)

**Comment by antuz123** [comment]
Couple more on the dashboard.

**Comment by AliMeer** [comment]
This is for bl version 0.83.0 and the recomendation is to upgrade to v>1.2.3. We have bl as a dependency of `react-native-level-fs` which is used by RN. We will need to wait for an updated version of `react-native-level-fs` or its dependency levelup to be available before bl version can be upgraded.

**Comment by AliMeer** [comment]
The other alert is about `semver` and the recommendation is to upgrade to v.4.3.2. we have a lower version of semver as a dependency of `levelup` which is a dependency of ` react-native-level-fs` which is used by RN.
We will need to wait for  `react-native-level-fs ` to be upgraded before this can be addressed.

**Comment by antuz123** [comment]
Can you create specific issues then for these? We can discuss where to place them based on the RN dependencies. @AliMeer

**Comment by AliMeer** [comment]
> Can you create specific issues then for these? We can discuss where to place them based on the RN dependencies. @AliMeer 

- Actually both are going to be addressed when` react-native-level-fs` is upgraded there is already a PR in `react-native-level-fs` to address this. Once the PR is merged and available both alerts will be addressed.
- We don't really have an impact due to these alerts as we don't use the the modules directly and the impacted code is not used in our project.

**Comment by AliMeer** [comment]
Verified on the repo page, there isn't any further update available for `levelup` will need to wait for an update in the meantime this is not a serious concern and can be postponed.

**Comment by AliMeer** [comment]
There is still no further update on this from the repo maintainers and the PR is still open

**Comment by AliMeer** [comment]
When this PR is approved and merged https://github.com/tradle/react-native-level-fs/pull/2
We can upgrade and close this issue

**Comment by ASN-BitHyve** [comment]
Still relevant or to be closed? @antuz123

**Comment by Deveshshankar** [comment]
Not relevant
